app/forms.py

```diff
@@ -0,0 +1,27 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField, BooleanField, RadioField, SelectField, TextAreaField, IntegerRangeField, DecimalRangeField
```

Comment:
not sure if wtforms.DecimalRangeField is a necessary import. maybe you forgot to remove it?